slides/02-data-budget.qmd

```diff
@@ -282,27 +292,29 @@ ggplot(frog_train, aes(age, latency, color = reflex)) +
 
 ```{r echo = FALSE}
 #| fig-align: 'center'
-quartiles <- quantile(frog_train$latency, probs = c(1:3)/4)
-ggplot(frog_train, aes(latency)) +
+quartiles <- quantile(taxi_train$distance, probs = c(1:3)/4)
+
+ggplot(taxi_train, aes(distance)) +
   geom_histogram(bins = 20) +
+  facet_grid(vars(tip)) +
   geom_vline(xintercept = quartiles, color = train_color, 
-             size = 1.5, lty = 2)
+             linewidth = 1.5, lty = 2)
 ```
 
 Stratified sampling would split within each quartile
 
 :::notes
-Based on our exploration, we realized that stratifying by latency might help get a consistent distribution. For instance, we'd include high and low latency in both the test and training
+Based on our exploration, we realized that stratifying by distance might help get a consistent distribution. For instance, we'd include high and low distance in both the test and training
 :::
 
 ## Stratification
 
-Use `strata = latency`
+Use `strata = distance`
```

Comment:
Really what would be helpful here is a spline term, but there does seem to be an effect of a smaller proportion of middle-distance trips resulting in a tip.

Reply:
Do we maybe want to stratify based on the outcome instead?

Reply:
Yes, thank you! [e35c41b](https://github.com/tidymodels/workshops/pull/108/commits/e35c41b98a0a91a1d4d3b54be04755f5962af808)